packages/circuit-ui/components/AutocompleteInput/AutocompleteInputService.ts

```diff
@@ -42,3 +60,20 @@ export const isOptionFocused = (
   activeOption?: number,
 ) =>
   activeOption !== undefined ? optionValues[activeOption] === value : false;
+
+export const updateMultipleSelectionValue = (
+  currentValue: AutocompleteInputOption[],
+  value?: AutocompleteInputOption,
```

Comment:
When would `value` be undefined? And is should we really return an empty array instead of the current values in this case?

Reply:
These typings might seem equivalent (both allow `undefined` values). However, the suggested code will show an error if a developer forgets to pass the `value` arg while still allowing it to have an `undefined` value.

```suggestion
  value: AutocompleteInputOption | undefined,
```